### PR TITLE
Reduce batch size to fit T5 paper

### PR DESCRIPTION
**Pull request by thomasw21**
One of the things we want to understand is why does out model seem to underperform?

One thing we noticed is that learning rate/learning rate schedule is extracted from the t5 paper, but we've update the batch size to fit with Meg-DS (ie 2048 instead of 128)?

Typically learning rate is constant for the first 1e4 steps, however since t5 starts decaying at 128 * 1e4 * 512 = 655M tokens whereas we start decaying at 2048 * 1e4 * 512 = 10B tokens (which is about 1/3 of the entire training data). Can this explain the poor performances? If so, what do we so? Reduce batch size to 128? Tune the learning_rate_scheduler.warmup_steps to be 1e4 / (2048 / 128) = 6250 . I'd be in favor of reducing batch size to mimick T5 paper.

In this PR I suggest reducing the batch size back to the original paper.

### CODE REVIEW

**Review by TevenLeScao — APPROVED** [review]

**Comment by craffel** [comment]
I would suggest reverting this change - a batch size of 2048 should work fine (it's what we used for the final T5 models). Adafactor is roughly invariant to batch size.

**Comment by thomasw21** [comment]
Reverted in bb34fda312455b774d5ca1d38783ad37d88b4a6b

**Comment by thomasw21** [comment]
Re applied the change as we've seen promising results with this modification.
